packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1741,7 +1745,7 @@ export class GLTFLoader implements IGLTFLoader {
 
                 if (outputOffset > 0) {
                     const name = `${animation.name || `animation${animation.index}`}_channel${channel.index}_${numAnimations}`;
-                    property.buildAnimations(targetInfo.target, name, fps, keys, (babylonAnimatable, babylonAnimation) => {
+                    propertyInfo.buildAnimations(target, name, fps, keys, (babylonAnimatable: IAnimatable, babylonAnimation: Animation) => {

```

Comment:
Why do you need to specify the types now?

Reply:
I think it was showing a linting error here, but it went away